x-pack/test/usage_collection/test_suites/stack_management_usage/index.ts

```diff
@@ -10,15 +10,15 @@ import { stackManagementSchema } from '@kbn/kibana-usage-collection-plugin/serve
 import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
-  // FLAKY: https://github.com/elastic/kibana/issues/119038
-  describe.skip('Stack Management', function () {
+  describe('Stack Management', function () {
     const { common } = getPageObjects(['common']);
     const browser = getService('browser');
 
     let registeredSettings: Record<string, any>;
 
     before(async () => {
       await common.navigateToApp('home'); // Navigate to Home to make sure all the appIds are loaded
+      await common.isChromeVisible(); // Make sure the page is fully loaded
```

Comment:
After going through the [internals](https://github.com/elastic/kibana/blob/main/test/functional/page_objects/common_page.ts#L219) of `common.navigateToApp`, I'm not surprised we're starting to see flakiness. `navigateToApp` does a whole lot before finally checking if chrome is loaded. However, `isChromeVIsible` assumes chrome is loaded if the globalNav itself is loaded, so we're still making assumptions.

I'm not exactly sure why the fix in https://github.com/elastic/kibana/pull/136808 didn't sort this test out, unless we're really encountering a race condition in the test itself.